source/rc.php-fpm/php-fpm/dependencies

```diff
@@ -0,0 +1,5 @@
+bind-run
+# Comment out "remount-root" and uncomment "filesystems-local"
+# if you use separate partition for /var/log
```

Comment:
A minor change, this should be just "/var" instead of "/var/log" :D

Reply:
Changed to `/var`.